pkgs/applications/science/logic/isabelle/default.nix

```diff
@@ -1,24 +1,35 @@
-{ lib, stdenv, fetchurl, perl, perlPackages, makeWrapper, nettools, java, polyml, z3, rlwrap }:
+{ lib, stdenv, fetchurl, callPackage, runCommand, coreutils, perl, perlPackages
+, makeWrapper, nettools, java, polyml, z3, rlwrap, naproche }:
```

Comment:
Are `perlPackages` and `makeWrapper` being used?

It doesn't seem that they are needed... `makeWrapper` is added as a dependency but then `makeWrapper`/`wrapProgram` is not called anywhere?

Reply:
Both `perlPackages` and `makeWrapper` are used here:

https://github.com/NixOS/nixpkgs/blob/c50f25092c20b7f8ae6bebd21d931963b5fc5b1f/pkgs/applications/science/logic/isabelle/default.nix#L96